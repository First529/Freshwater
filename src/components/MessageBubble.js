import React from "react";

export default class MessageBubble extends React.Component {
  render() {
    return (
      <div className="message">
        <div className="message-text">{this.props.text}</div>
        <div className="message-time">{this.props.time}</div>
      </div>
    );
  }
}
