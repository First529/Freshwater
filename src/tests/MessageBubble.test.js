import React from "react";
import { shallow } from "enzyme";
import MessageBubble from "../components/MessageBubble";

it("Get sender message correctly"),
  () => {
    const wrapper = mount(
      <MessageBubble text={"Hello World"} time={"15:30"} />
    );

    expect(wrapper.find(".message-bubble")).toBeDefined();
    expect(wrapper.find(".message-time")).toBeDefined();
  };
