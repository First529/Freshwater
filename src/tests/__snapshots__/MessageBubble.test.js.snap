// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageBubble render correctly 1`] = `
<div>
  <div
    className="message-bubble"
  />
  <div
    className="message-time"
  />
</div>
`;

